assets/src/components/v2/elevator/no_data.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ComponentType } from "react";
+import { imagePath } from "Util/util";
+
+const NoData: ComponentType = () => {
+  return (
+    <>
+      <img src={imagePath(`elevator-status-no-data.png`)} />
+    </>
```

Comment:
This enclosing fragment should be unneeded, since it only contains a single element.